src/Libraries/Microsoft.Extensions.Http.Resilience/Microsoft.Extensions.Http.Resilience.csproj

```diff
@@ -38,4 +38,47 @@
     <InternalsVisibleToTest Include="$(AssemblyName).Tests" />
     <InternalsVisibleToDynamicProxyGenAssembly2 Include="*" />
   </ItemGroup>
+
+  <PropertyGroup>
+    <!-- Defines whether we need to pack buildTransitive artifacts for net462. -->
+    <PackNet462 Condition=" '$(MinimumSupportedTfmForPackaging)' != '' and
+                            '$(ConditionalNet462)' != '' and
+                            ($(TargetFrameworks.Contains('$(MinimumSupportedTfmForPackaging)')) and $(TargetFrameworks.Contains('$(ConditionalNet462)'))) "
+                >true</PackNet462>
+  </PropertyGroup>
```

Comment:
It's a good idea to create a variable to avoid duplications. 👍 

Though the variable needs to live in eng/MSBuild/Packaging.props.
This way we can update use it here and in 
https://github.com/dotnet/extensions/blob/f335f2a9928580a361807ba2ced86f1ed27aca0b/eng/MSBuild/Packaging.targets#L20-L22

Reply:
Thank you. Moved the variable to the Packaging.targets file as you'd suggested.